<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>ISOCP2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>计算器by娄卫健</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-color: rgb(29,28,28);
    color: rgb(214,214,214) ;
    font-size:25px ;
 	border:none;
	border-radius:25px
}
QWidget{border:none;
}</string>
  </property>
  <widget class="QPushButton" name="bt7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btjia">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: rgb(255,121,32) ;}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btjian">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: rgb(255,121,32) ;}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btcheng">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: rgb(255,121,32) ;}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btchu">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {color: rgb(255,121,32) ;}</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btdeng">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(254,118,26);
    color: rgb(213,213,213) ;
    font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt0">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btdian">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xianshi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background-color: rgb(13,13,13);
	color: rgb(184,184,184) ;
    font-size:30px;
    border:1px solid rgb(31,31,31);

}</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttui">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QpushiButton{background-image:url(button.png);
background-color:none}</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btyan">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-image: url(sqrt2.png);
	color:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="beijing">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-image: url(back3.png);
	color: rgb(182,182,182) ;
    font-size:30px;
	border:none;

}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btmenu">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-image:url(back4.png);
color: rgb(255,121,32) ;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="beijing2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>21</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background-color: rgb(13,13,13);
	color: rgb(79,79,79) ;
    font-size:30px;
    border:none;

}</string>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btans">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btjue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>	  |x|  |</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btsin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btping">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string> x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btcos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btsan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string> x³</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttan">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btjie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string> x!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bty">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btz">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xianshi2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background-color: rgb(13,13,13);
	color: rgb(184,184,184) ;
    font-size:30px;
    border:1px solid rgb(31,31,31);

}</string>
   </property>
   <property name="maxLength">
    <number>18</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btshu">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btxian">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btrt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>rt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btpan">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>pan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btm1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>M+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btm2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>M-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btm3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btm4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size:18px}</string>
   </property>
   <property name="text">
    <string>解方程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt3_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="beijing1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>641</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {background-color: rgb(21,21,21);
	color: rgb(182,182,182) ;
    font-size:30px;
	border:2px solid rgb(31,31,31);

}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>beijing1</zorder>
  <zorder>xianshi</zorder>
  <zorder>bt7</zorder>
  <zorder>bt4</zorder>
  <zorder>bt1</zorder>
  <zorder>bt8</zorder>
  <zorder>bt5</zorder>
  <zorder>bt2</zorder>
  <zorder>bt9</zorder>
  <zorder>bt6</zorder>
  <zorder>bt3</zorder>
  <zorder>btjia</zorder>
  <zorder>btjian</zorder>
  <zorder>btcheng</zorder>
  <zorder>btchu</zorder>
  <zorder>btdeng</zorder>
  <zorder>bt0</zorder>
  <zorder>btdian</zorder>
  <zorder>bttui</zorder>
  <zorder>beijing</zorder>
  <zorder>beijing2</zorder>
  <zorder>btyan</zorder>
  <zorder>btans</zorder>
  <zorder>btjue</zorder>
  <zorder>btsin</zorder>
  <zorder>btping</zorder>
  <zorder>btcos</zorder>
  <zorder>btsan</zorder>
  <zorder>bttan</zorder>
  <zorder>btjie</zorder>
  <zorder>bty</zorder>
  <zorder>btz</zorder>
  <zorder>btc</zorder>
  <zorder>btmenu</zorder>
  <zorder>xianshi2</zorder>
  <zorder>btshu</zorder>
  <zorder>btxian</zorder>
  <zorder>btrt</zorder>
  <zorder>btpan</zorder>
  <zorder>btm1</zorder>
  <zorder>btm2</zorder>
  <zorder>btm3</zorder>
  <zorder>btm4</zorder>
  <zorder>bts</zorder>
  <zorder>bt3_7</zorder>
  <zorder>toolButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
